Awesome books: a complete website with navigation

Nedramrez opened the pull request:
- Used JavaScript to manipulate DOM elements.
- Used JavaScript events
- Modified the Awesome books application to have:
    A Navigation bar.
Three content sections:
Books list.
Add book form.
Contact info.

ometman committed:
Add modules for navigation

Nedramrez committed:
Install luxon

ometman committed:
Add script for content change

Nedramrez committed:
Add an addbook-event.js and footer.js files.

ometman committed:
Add book class, book collection and contact js

Nedramrez committed:
Change style according to the wireframe.

ometman committed:
Add logo

ometman committed:
Fix linters errors

usorfaitheloho commented on modules/luxon.js:
```diff
@@ -0,0 +1,7265 @@
+
+export {
+  DateTime, Duration, FixedOffsetZone, IANAZone, Info, Interval, InvalidZone, Settings, SystemZone, VERSION, Zone,
+};
```
You did well in installing the luxon module for date and time configuration. However it doesn't seem to function as it shown in the design given. Please kindly implement this so that your design replicates the original design 😄 

![image](https://github.com/ometman/awesomebooks-omet-zerma/assets/20537712/8954d333-abc0-43db-bcf3-91abdef13d35)

usorfaitheloho commented on index.css:
```diff
@@ -1,103 +1,240 @@
-  align-items: center;
-  width: 100vw;
-  font-family: 'Arial', sans-serif;
+  font-family: c, sans-serif;
+  font-family: 'Lato', sans-serif;
+  font-size: 16px;
+  max-width: 1200px;
+  height: 100vh;
+  margin: auto;
+  padding: 0;
+  background-image: url(./book-bg.jpg);
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
```
Kindly reduce the fontsize for your project so it looks similar to the design given 👍

usorfaitheloho reviewed (changes requested):
> Hi @ometman  and @Nedramrez  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

 ![](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966) [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * All linter checks passed ✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Nedramrez committed:
Reviewer's demand:
1. Change CSS to make date and time appear at the website.
2. Change the font size.

Nedramrez committed:
Reviewer's demand:
1. Center the contact info.

AbelG101 reviewed (approved):
![](https://media.giphy.com/media/eKfovWZOlOpbTDEX9F/giphy.gif)

# Hi @Nedramrez 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- Navigation works properly ✔
- UI matches wireframe ✔
- Correct use of GitHub Flow ✔
- All linter checks are passing ✔
- Descriptive PR title with a good summary ✔

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
